src/components/metadata/CollectionMetadata.vue

```diff
@@ -24,10 +24,16 @@ import MetadataItem from '@/components/metadata/MetadataItem.vue';
 
 const contentStore = useContentsStore();
 
-function getCollectorName(collection: Collection) : string | null {
+function getCollectorName(collection: Collection) : string[] | null {
   if (!collection) return null;
   if (collection.collector.length === 0) return null;
-  return collection.collector[0].name;
+  if (collection.collector.length === 1) return [collection.collector[0].name];
+
+  let collectors: string[] = []
+  collection.collector.forEach((collector) => {
```

Comment:
To simplify this, you could use `.map` function. Also the return type should stay the same (string). This can be achieved by using `.join(', ')`after `map` with an inline return. Please pay attention to the space after comma.